### PR TITLE
Logging:Log in added using logback in flexible helloworld example

### PR DESCRIPTION
**Pull request by abhinav-qlogic**
Fixes #3510

### CODE REVIEW

File: flexible/helloworld/src/main/java/com/example/flexible/helloworld/HelloServletFilter.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+@WebFilter(filterName = "HelloServletFilter")
+public class HelloServletFilter implements Filter {
+
+  private final Logger logger = LoggerFactory.getLogger(HelloServletFilter.class);
+
+  @Override
+  public void init(FilterConfig filterConfig) {
+    try {
+      LoggerContext context = (LoggerContext) LoggerFactory.getILoggerFactory();
```
Comment (lesv) [inline flexible/helloworld/src/main/java/com/example/flexible/helloworld/HelloServletFilter.java]: I wonder if some comments might be helpful here.

**Comment by lesv** [comment]
Just comment back if your ok w/ this as is.  It still has to pass tests.

File: flexible/helloworld/src/main/java/com/example/flexible/helloworld/HelloServletFilter.java (hunk 1 above)
Comment (abhinav-qlogic) [reply flexible/helloworld/src/main/java/com/example/flexible/helloworld/HelloServletFilter.java]: done, please let me know anything else required to add.

**Comment by kurtisvg** [comment]
Closing this as it has gone stale. Please reopen if you are still interested in PRing a change.

**Pull request closed without merging**